backend/dataall/core/environment/api/resolvers.py

```diff
@@ -677,7 +678,7 @@ def get_pivot_role_name(context: Context, source, organizationUri=None):
             resource_uri=organizationUri,
             permission_name=permissions.GET_ORGANIZATION,
         )
-        pivot_role_name = SessionHelper.get_delegation_role_name()
+        pivot_role_name = SessionHelper.get_delegation_role_name(region='<REGION>')
```

Comment:
when a user copies pivotrole name it will now show as `dataallPivotRole-<REGION>` on link env UI yes?

Reply:
There are multiple scenarios for pivot roles:
- manual pivot role -->`get_delegation_role_name` returns `dataallPivotRole`
- CDK SINGLE pivot role, `core.features.cdk_pivot_role_multiple_environments_same_account` is set to False -->`get_delegation_role_name` returns `dataallPivotRole-cdk` 
- CDK MULTIPLE pivot roles, `core.features.cdk_pivot_role_multiple_environments_same_account` is set to True -->`get_delegation_role_name` returns `dataallPivotRole-cdk-<REGION>` 

The resolver `get_pivot_role_name` is called in `getPivotRoleName` query that is only used in the EnvironmentCreateForm. For CDK created roles, the users do not even see any name in the UI, but I left it coded in case we want to plot anything in the UI down the line. For the manual pivot role they will be able to see and copy `dataallPivotRole`